build.gradle

```diff
@@ -3,3 +3,22 @@ plugins {
     id 'io.micronaut.build.internal.dependency-updates'
     id 'io.micronaut.build.internal.quality-reporting'
 }
+
+repositories {
+    mavenCentral()
+    maven { url "https://s01.oss.sonatype.org/content/repositories/snapshots/" }
+}
+
+// This should be removed when the build plugins change to use java 11/17
+// Neo4j 4.x requires Java 11, and javadoc fails when it uses Java 8
+// with error: cannot access GraphDatabaseSettings...class file has wrong version 55.0, should be 52.0
+java {
```

Comment:
In the GCP project, which is only compatible to 11, we set the following in the [root build file](https://github.com/micronaut-projects/micronaut-gcp/blob/master/build.gradle#L7-L10)

```groovy

micronautBuild {
    sourceCompatibility = "11"
    targetCompatibility = "11"
}
```